diff --git a/node_modules/istanbul-reports/lib/text/index.js b/node_modules/istanbul-reports/lib/text/index.js
index 538242a..4c049fd 100644
--- a/node_modules/istanbul-reports/lib/text/index.js
+++ b/node_modules/istanbul-reports/lib/text/index.js
@@ -187,7 +187,7 @@ function tableRow(
 
 function TextReport(opts) {
     opts = opts || {};
-    this.file = opts.file || null;
+    this.file = opts.file || 'coverage-final.txt'; // ... this is HORRIBLE, but I just can't deal with this terrible API any more
     this.maxCols = opts.maxCols || 0;
     this.cw = null;
     this.skipEmpty = opts.skipEmpty;
