diff --git a/node_modules/@nivo/bar/cjs/nivo-bar.js b/node_modules/@nivo/bar/cjs/nivo-bar.js
index 2caa4fd..cad4135 100644
--- a/node_modules/@nivo/bar/cjs/nivo-bar.js
+++ b/node_modules/@nivo/bar/cjs/nivo-bar.js
@@ -345,7 +345,7 @@ var generateVerticalStackedBars = function generateVerticalStackedBars(_ref) {
                     barHeight -= innerPadding;
                 }
 
-                if (barHeight > 0) {
+                if (barHeight >= 0) { // Patched to fix https://github.com/plouc/nivo/issues/448
                     var barData = {
                         id: stackedDataItem.key,
                         value: d.data[stackedDataItem.key],
@@ -447,7 +447,7 @@ var generateHorizontalStackedBars = function generateHorizontalStackedBars(_ref2
                     barWidth -= innerPadding;
                 }
 
-                if (barWidth > 0) {
+                if (barWidth >= 0) { // Patched to fix https://github.com/plouc/nivo/issues/448
                     bars.push({
                         key: stackedDataItem.key + '.' + index,
                         data: barData,
