diff --git a/node_modules/@types/jsurl/index.d.ts b/node_modules/@types/jsurl/index.d.ts
index d394057..22fc2d5 100755
--- a/node_modules/@types/jsurl/index.d.ts
+++ b/node_modules/@types/jsurl/index.d.ts
@@ -3,6 +3,8 @@
 // Definitions by: Alexey Gorshkov <https://github.com/agorshkov23>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+export = JSURL;
+
 interface UrlQuery {
     clear: () => void;
 }
